src/shared/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,13 @@
+import { CheckboxProps } from "@mui/material";
+import * as S from "./styles";
+
+type Props = CheckboxProps;
+
+export const CheckBox: React.FC<Props> = ({ ...props }) => {
+  return (
+    <S.Container>
+      <S.Label control={<S.Wrapper {...props} />} label={props["aria-label"]} />
+    </S.Container>
```

Comment:
Alterar o nome dos component para seguir o padrão utilizado no projeto:

```suggestion
    <S.Wrapper>
      <S.Label control={<S.CheckboxControl {...props} />} label={props["aria-label"]} />
    </S.Wrapper>
```